app/src/androidTest/java/sdp/moneyrun/MenuActivityTest.java

```diff
@@ -86,5 +86,9 @@ public void logOutButtonWorks() throws InterruptedException {
         assertEquals(State.DESTROYED, testRule.getScenario().getState());
     }
 
+    @Test
+    public void checkIfSetPlayerObjectSetsThePlayerCorrectly(){
+
```

Comment:
Why the  empty test ?

Reply:
Because I forgot to write it apparently, thanks will do it right away

Reply:
Actually the method I wanted to test was never used so there was no point in testing it, still I will remove the test in that case